File: node/controllers/delete-unit-from-roster-controller.ts
import { Request, Response } from 'express';
import { deleteUnitFromArmyRosterFromDb } from '../models/delete-unit-from-army-roster-model';

export async function deleteUnitFromArmyRosterController(req: Request, res: Response) {
    try {
        const roster_unit_id = Number(req.body.roster_id) || 0;
        const unitData = await deleteUnitFromArmyRosterFromDb(roster_unit_id);
        res.statusCode = 200;
        res.json(unitData);
    } catch (error) {
        res.statusCode = 500;
        res.send();
    }
}

File: python/controllers/delete_unit_from_roster_controller.py
from flask import request

from models.delete_unit_from_roster_model import delete_unit_to_roster_model


def delete_unit_roster_controller():
    body = request.get_json()
    if body is None:
        return []
    roster_id = body.get("roster_id", 0)
    if roster_id is 0:
        return []
    return delete_unit_to_roster_model(roster_id)
